docs/streams/stream-examples/realtime-updates.md

```diff
@@ -74,8 +105,172 @@ async function createCollection () {
+    if client.has_collection(COLLECTION_NAME):
+        collection = client.collection(COLLECTION_NAME)
+    else:
+        collection = client.create_collection(COLLECTION_NAME, stream=True)
+
+    # Subscribe to be notified when changes are made to collection.
+
+    def create_callback():
+        def callback_fn(event):
+            pp.pprint(event)
+            return
+
+        client.on_change(COLLECTION_NAME, callback=callback_fn, timeout=15)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 4. Add Data and Subscribe to Collection
+
+Add to the collection the information we defined in the `data` variable, then subscribe to the collection.
+
+In this example, we are adding IP addresses to blacklist.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
 async function insertData () {
-  console.log(`\n3. Insert data in region ${globalUrl}`);
+  console.log(`\n3. Insert data`);
+  await client.insertDocumentMany(collectionName, data);
+}
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step3: Subscribe to receive documents in realtime (PUSH model)
+    print(f"\n3. SUBSCRIBE_COLLECTION: region: {URL},  collection: {COLLECTION_NAME}")
+    rt_thread = threading.Thread(target=create_callback)
+    rt_thread.start()
+    time.sleep(10)
+    print(f"Callback registered for collection: {COLLECTION_NAME}")
+
+    # Step4: Subscribe to receive documents in real-time (PUSH model)
+    print(f"\n4. INSERT_DOCUMENTS: region: {URL},  collection: {COLLECTION_NAME}")
+    client.insert_document(COLLECTION_NAME, document=data)
+
+    # Step5: Wait to close the callback.
+    print("\n5. Waiting to close callback")
+    rt_thread.join(2)
+```
+
+</TabItem>
+</Tabs>
+
+
+### Step 5. Delete Data
+
+Delete the data from the collection. If the collection becomes empty, the collection is deleted too.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+async function deleteData () {
+  console.log("\n4. Delete data");
+  await client.deleteCollection(collectionName);
+}
+(async function () {
+  await createCollection();
+  await sleep(2000);
+  await insertData();
+  await sleep(10000);
+  await listener.close();
+  await deleteData();
+})();
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+
+```py
+if __name__ == '__main__':
+
+    # Step 6: Delete data.
+    print(f"\n6. DELETE_DATA: region: {URL}, collection: {COLLECTION_NAME}")
+    collection.truncate()
+    client.delete_collection(COLLECTION_NAME)
+```
+
+</TabItem>
+</Tabs>
+
+
+## Full Demo File
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+const jsc8 = require("jsc8");
+
+const client = new jsc8({url: "https://play.paas.macrometa.io", apiKey: "XXXX", fabricName: '_system'});
+
+// Variables
+const collectionName = "ddos";
+let listener;
+const data = [
+  { ip: "10.1.1.1", action: "block", rule: "blocklistA" },
```

Comment:
Avoid "blacklist" and "whitelist" in all cases. Use blocklist and allowlist instread.